lxd-migrate/transfer.go

```diff
@@ -131,6 +132,28 @@ func rsyncSendSetup(ctx context.Context, path string, rsyncArgs string, instance
 	return cmd, conn, stderr, nil
 }
 
+func sendBlockVol(ctx context.Context, conn io.WriteCloser, path string) error {
```

Comment:
I would have expected this function to just use the path arg directly, so it can be used with block devices and image files. Adding `root.img` to the path seems unexpected.